GrowthBook/src/commonMain/kotlin/com/sdk/growthbook/features/FeaturesViewModel.kt

```diff
@@ -56,6 +56,20 @@ internal class FeaturesViewModel(
                         isRemote = false
                     )
                 }
+                dataModel.encryptedFeatures?.let { encryptedFeatures: String ->
+                    encryptionKey?.let { encryptionKey ->
+                        val features = getFeaturesFromEncryptedFeatures(
+                            encryptedString = encryptedFeatures,
+                            encryptionKey = encryptionKey,
+                        )
+                        features?.let {
+                            this.delegate.featuresFetchedSuccessfully(
```

Comment:
This may override `dataModel.features` that we restored previously (on line 54). 

Should we add the same check that we do in `prepareFeaturesData`? 
```kotlin
if (!features.isNullOrEmpty()) {
  // restore features from disk
} else { 
  if (encryptedFeatures != null && encryptionKey != null) { 
    // restore encrypted features from disk
  } 
}
```

Reply:
If encryption is disabled, `encryptionKey` will be `null` `featuresFetchedSuccessfully()` will not be called the second time.